pkg/apiserver/resthandler.go

```diff
@@ -470,11 +470,11 @@ func PatchResource(r rest.Patcher, scope RequestScope, typer runtime.ObjectTyper
 		ctx := scope.ContextFunc(req)
 		ctx = api.WithNamespace(ctx, namespace)
 
-		// PATCH requires same permission as UPDATE
-		if admit.Handles(admission.Update) {
+		if admit != nil && admit.Handles(admission.Patch) {
 			userInfo, _ := api.UserFrom(ctx)
 
-			err = admit.Admit(admission.NewAttributesRecord(obj, scope.Kind.GroupKind(), namespace, name, scope.Resource.GroupResource(), scope.Subresource, admission.Update, userInfo))
+			// TODO this record should include the patch itself.  Right now we can't do that because patches are not runtime.Objects
```

Comment:
> In that case-- it's currently broken now, in that you cannot forbid transitions, correct? So leaving this out doesn't make it more broken. I would much rather fix the update to take old & new. Patch is an implementation detail and the admission/authorization machinery should work exactly the same between patch and update.

Would simply removing this line satisfy you for now?  I'd rather not drift too far from upstream, but refactoring admission plugins isn't something I plan to tackle now.